src/derivative_operators/abstract_operator_functions.jl

```diff
@@ -190,6 +185,19 @@ end
 function DiffEqBase.update_coefficients!(A::AbstractDerivativeOperator,u,p,t)
     if A.coeff_func !== nothing
         A.coeff_func(A.coefficients,u,p,t)
+    else
+        @warn "No coeff_func found. No updating performed."
+    end
+end
+function DiffEqBase.update_coefficients!(A::AbstractDerivativeOperator) where {T<:Number}
```

Comment:
This one doesn't make sense: it's only in the API with `(u,p,t)`, and this wouldn't be widely usable.

Reply:
I agree. I put there in case it maybe was needed, but I'm happy to delete these lines.